gopom.go

```diff
@@ -24,6 +24,37 @@ func Parse(path string) (*Project, error) {
 	return &project, nil
 }
 
+func ParseFromReader(reader io.Reader) (*Project, error) {
+	b, _ := ioutil.ReadAll(reader)
+	var project Project
+
+	err := xml.Unmarshal(b, &project)
+	if err != nil {
+		return nil, err
+	}
+	return &project, nil
+}
+
+func ParseFromByte(b []byte) (*Project, error) {
+	var project Project
+
+	err := xml.Unmarshal(b, &project)
+	if err != nil {
+		return nil, err
+	}
+	return &project, nil
+}
+
+func ParseFromString(s string) (*Project, error) {
```

Comment:
Similar to the above comment.
We can construct a io.Reader from string through 
`strings.NewReader(b)` which means we can instead use the `ParseFromReader` function.